packages/legacy/core/App/types/help.ts

```diff
@@ -0,0 +1,10 @@
+import { StackNavigationProp } from '@react-navigation/stack'
```

Comment:
Same with this file name. "help" is somewhat generic unless its for all app help.

Reply:
Changed